scripts/darwin/build.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+
+# Load system env only
+source /etc/bashrc
+source /etc/profile
+
+# Make sure home is pointing to current directory
+export HOME=$(pwd)
+# Calculate number of available cores
+export J=$(( $(nproc --all) )) && echo Using ${J} cores during build
+
+COMPILER_MODULE=${15}
+MPI_MODULE=${17}
+
+export TMPDIR=${HOME}/tmp
+
+# Temp directory apparently needed by spack because of OSError: [Errno 18] Invalid cross-device link
+if [ -d ${TMPDIR} ] 
+then
+  echo "Removing ${TMPDIR}"
+  rm -rf ${TMPDIR}/*
+  rmdir ${TMPDIR}
```

Comment:
```suggestion
  rm -rf ${TMPDIR}
```
Otherwise the second command my fail if there are hidden files and directories in the tmp folder.